frontend/www/js/omegaup/components/contest/NewForm.vue

```diff
@@ -497,23 +489,24 @@ export default class NewForm extends Vue {
     this.$emit('create-contest', request);
   }
 
-  blockCatLanguage(): void {
-    for (const problem of this.initialProblems) {
-      if (problem.languages.split(',').includes('cat')) {
-        this.catLanguageBlocked = true;
-        break;
+  get catLanguageBlocked(): boolean {
+    if (this.problems) {
```

Comment:
```suggestion
    if (!this.problems) {
      return false;
    }
```

más lugares donde se puede usar el guard clause pattern!